README.md

```diff
@@ -55,15 +56,20 @@
 - [neotracker.io](https://neotracker.io/) - MainNet and TestNet explorer, created by the Neo Tracker team.
 - [explorer.neoverse.io](http://explorer.neoverse.io/) - Explorer created by the NEOverse team.
 
+## Blockchain APIs
+
+* [Neon wallet official REST api](https://github.com/CityOfZion/neon-wallet-db#overview-of-api)
```

Comment:
Please add a short description, after the URI.

Reply:
Please add in that description that the overview is partially outdated. It mixes `/v1/` and `/v2` api calls in the description and examples, while they all have a working `/v2/` version.